exonum-java-binding-time-service/src/main/java/com/exonum/binding/time/TimeSchemaImpl.java

```diff
@@ -24,22 +24,28 @@
 import com.exonum.binding.storage.indices.ProofMapIndexProxy;
 import java.time.ZonedDateTime;
 
-public class TimeSchemaProxy implements TimeSchema {
+public class TimeSchemaImpl implements TimeSchema {
 
   private static final Serializer<PublicKey> PUBLIC_KEY_SERIALIZER = StandardSerializers.publicKey();
   private static final Serializer<ZonedDateTime> ZONED_DATE_TIME_SERIALIZER = StandardSerializers.zonedDateTime();
 
   private final View dbView;
 
-  private TimeSchemaProxy(View dbView) {
+  private TimeSchemaImpl(View dbView) {
+    checkIfEnabled();
```

Comment:
Will it work in ITs?

Reply:
1. In ITs of this class?
2. In ITs that happen to use TimeSchema?

Reply:
Will fix after native part implementation

Reply:
But what is the specified behaviour? It can't be arbitrary at this moment.